name-registry/tests/functions/expiry.rs

```diff
@@ -0,0 +1,41 @@
+mod success {
+    use crate::utils::{
```

Comment:
If the same imports are being used then I would factor them out to be on the top level and then perhaps do specific imports in each module

Reply:
Using super just adds more lines for no reason (If you import at the top level the mods dont get access to the import, so you have to use super to import them again, making the top level import kinda pointless)